source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -559,14 +559,14 @@ void ContextImpl::logHandshake(SSL* ssl) const {
     stats_.no_certificate_.inc();
   }
 
-#ifndef BORINGSSL_FIPS
+#if BORINGSSL_API_VERSION >= 18
```

Comment:
While you're at it, can you add the following so we remember to delete this when fips is current enough?

```
#if defined(BORINGSSL_FIPS) && BORINGSSL_API_VERSION >= 18
#error "Delete this block; no longer needed"
#endif
```

Or something to that effect?